src/taco-dependency-installer/test/tacoErrors.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * ******************************************************
+ *                                                       *
+ *   Copyright (C) Microsoft. All rights reserved.       *
+ *                                                       *
+ *******************************************************
+ */
+
+/// <reference path="../../typings/mocha.d.ts" />
+
+"use strict";
+
+import mocha = require ("mocha");
+import path = require ("path");
+
+import resources = require ("../resources/resourceManager");
+import testUtil = require ("../../taco-utils/testUtils/testHelper");
```

Comment:
Agreed. It would be better to make sure that taco-utils is a dependency in package.json, and then `require("taco-utils/testUtils/testHelper")` without the relative path to refer to the particular file. Not sure whether that will provide intellisense/typings appropriately though.